shared/teams/confirm-modals/index.stories.tsx

```diff
@@ -1,13 +1,19 @@
 import * as React from 'react'
 import * as Sb from '../../stories/storybook'
+import * as Container from '../../util/container'
 import ConfirmKickOut from './confirm-kick-out'
+import DeleteChannel from './delete-channel'
 import {store, fakeTeamID} from '../stories'
 import reallyLeaveTeam from './really-leave-team/index.stories'
 
+const storeWithSelection = Container.produce(store, draftState => {
+  draftState.teams.teamSelectedChannels = new Map([[fakeTeamID, new Set(['1', '2'])]])
```

Comment:
ooh you may want to change to channel IDs in the other place where we edit draftState.teams.teamSelectedChannels in a story too while you're at it.